### PR TITLE
 Add Product List to API V2 docs

### PR DESCRIPTION
**Pull request by damianlegawiec**

### CODE REVIEW

**Comment by VygovskySergey** [comment]
Should we add to filtering, query by ids?

**Comment by damianlegawiec** [comment]
@VygovskySergey good idea - by IDs you mean product IDs, correct?

**Comment by VygovskySergey** [comment]
@damianlegawiec Yeah, that what I mean

**Comment by kushniryb** [comment]
@damianlegawiec 
This update introduces the following errors - https://d.pr/free/i/0MqTmF

**Review by kushniryb — CHANGES_REQUESTED** [review]
@damianlegawiec 
This update introduces following errors - https://d.pr/free/i/0MqTmF

**Comment by damianlegawiec** [comment]
@kushniryb fixed!

File: api/docs/v2/storefront/index.yaml [hunk 1]
```diff
@@ -157,6 +157,54 @@ paths:
...
+          name: sort
+          schema:
+            type: string
+          example: >-
+            -updated_at,price
+          description:  >-
+            Sort products based on:
+            <ul>
+              <li>price (ascending/descenging)</li>
+              <li>price (ascending/descenging)</li>
```
Comment (VygovskySergey) [inline api/docs/v2/storefront/index.yaml]: duplication

File: api/docs/v2/storefront/index.yaml [hunk 2]
```diff
@@ -157,6 +157,54 @@ paths:
...
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ProductsList'
+  '/products/{id}':
+    get:
+      description: >-
+        To view the details for a single product, make a request using that
+        product's permalink:<br /> <code>GET /api/v2/products/a-product</code>
+        <br /><br />You may also query by the product's id attribute:<br /><code>GET
+        /api/v1/products/1 </code> <br /><br />Note that the API will attempt a
```
Comment (VygovskySergey) [inline api/docs/v2/storefront/index.yaml]: `/api/v2/products/1`

**Review by VygovskySergey — APPROVED** [review]
LGTM

**Review by kushniryb — APPROVED** [review]
👍

**Pull request merged**